pydatalab/pydatalab/models/people.py

```diff
@@ -62,6 +62,27 @@ def add_missing_verification(cls, v):
         return v
 
 
+class DisplayName(str):
+    """
+    A constrained string less than 150 characters long
+    that cannot be empty and should contain only alphabetic characters and spaces.
+    """
+
+    max_length = 150
+
+    def __new__(cls, value):
+        if not value.strip():
+            raise ValueError("Display name cannot be empty.")
+
+        if not all(char.isalpha() or char.isspace() for char in value):
+            raise ValueError("Display name can only contain alphabetic characters and spaces.")
+
```

Comment:
These are fine for now, might be easier/more performant to write these constraints as a regexp instead (that we could then duplicate in the UI). Just a note to myself!